src/renderer/components/LandingPage.vue

```diff
@@ -2,6 +2,7 @@
   <div id="wrapper">
     <main>
       <h1>Hello world!</h1>  
+      <img style="width: 100px;" src="../assets/day.svg" />
```

Comment:
We really have to use an <img /> tag ? No way to use svg directly ?

Reply:
Didn't find an easy way to do it... 😢